main.py

```diff
@@ -70,7 +70,7 @@ def main():
                     **scenario_params,
                     experiment_path=experiment_path,
                     scenario_id=scenario_id + 1,
-                    n_repeat=i + 1
+                    repeat_tag=i + 1
```

Comment:
Hm in fact I changed it in scenario, because the parameter n_repeat (at Scenario object level), is only used for the save-name of the scenario. But in main, maybe n_repeat is better